plugins/easyimage/plugin.js

```diff
@@ -6,6 +6,141 @@
 ( function() {
 	'use strict';
 
+	var commandsToRefresh = [ 'easyimageFull', 'easyimageSide' ];
+
+	function isSideImage( widget ) {
+		return widget.element.hasClass( 'easyimage-side' );
```

Comment:
Let's extract class names into constants, so that it is not duplicated all over the place.

Reply:
It should be configurable, so maybe just extract it to config variables?

Reply:
Make sense we can use configurable classes from the very beginning.